install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -1,5 +1,9 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+  ack-4.8-kube-1.22-api-removals-in-4.9: |
+      Kubernetes 1.22 and thus OpenShift 4.9 remove several APIs which require admin consideration. Please see
+      the knowledge article https://access.redhat.com/articles/6329921 for details and instructions.
```

Comment:
Check the text pointing to the KCS.
/cc @sdodson
/cc @bparees 
/cc @wking

Reply:
LGTM, @wking can you finish peer review and we'll unhold this after we confirm the publishing date with @bergerhoffer next week?